src/pages/Settings/index.js

```diff
@@ -867,6 +864,8 @@ class Settings extends PureComponent /*:: <SettingsProps, SettingsState> */ {
               </nav>
               <section>
                 <section onChange={changeSettings}>
+                  <AddToHomeScreen />
+                  <hr />
```

Comment:
```suggestion
```

First time I have a look at this component, since it doesn't render on my system (tried Firefox and Chromium). It's unclear to me how this is useful, especially on the settings page.

Reply:
This is for PWA, with it you can add an icon of InterPro as an app in your mobile, see https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeinstallprompt_event

I do use it that way, but I wonder how many people have also installed like that. Maybe, having it in settings is hiding it a bit too much.

Reply:
Yes, I ended up on this page as well. Do we know how many users access InterPro from a smartphone or tablet? Because if nearly all users access InterPro using a web browser on a desktop/laptop, I'd say this component can be removed.

Reply:
14K in the last month, It is quite significant!

![Screenshot 2024-05-10 at 12 10 31](https://github.com/ProteinsWebTeam/interpro7-client/assets/46671268/9d054f1c-d54a-4769-8f73-b01765b3fbbe)

Reply:
OK, then this is something that we can keep. Not the scope of this PR, but maybe the component could be displayed somewhere with more visibility than the settings page (side menu?).